script.js

```diff
@@ -0,0 +1,75 @@
+var evenOdd;
+var myFuncCalls = 0;
+if (myFuncCalls%2===0){
+  evenOdd="X"
+  }
+else{
+  evenOdd="O"
+  }
```

Comment:
Remember that this even / odd determination needs to happen in each function below. Two ways you could go about it:
1) Copy /  paste it into every function (not great, lots of duplication)
2) Turn this into a function, and call the function in each `changeText1`, `changeText2` function, etc.